proto/cosmos/crypto/keyring/hd.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package cosmos.crypto.keyring;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/crypto/keyring";
```

Comment:
If I change go_package to "github.com/cosmos/cosmos-sdk/crypto/keyring/hd" and `make proto-gen`, the error is raised `2021/05/27 14:36:49 protoc-gen-gogo: error:inconsistent package import paths: "github.com/cosmos/cosmos-sdk/crypto", "github.com/cosmos/cosmos-sdk/crypto/keyring"`. How can i fix that ? is there any way to set go_package to ""github.com/cosmos/cosmos-sdk/crypto/keyring/hd"?

Reply:
If putting in the current keyring folder creates import cycle, then we can creata new `hd` folder in proto